plugins-scripts/check_uptime.pl

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/perl -w
```

Comment:
please use: `#!@PERL@ -w`

Reply:
Thanks for the hint. I've changed that line by accident in the previous commit.